addons/cargo/functions/fnc_onMenuOpen.sqf

```diff
@@ -22,37 +22,67 @@ params ["_display"];
 uiNamespace setVariable [QGVAR(menuDisplay), _display];
 
 if (GVAR(interactionParadrop)) then {
-    (_display displayCtrl 12) ctrlSetText (localize LSTRING(paradropButton));
+    (_display displayCtrl 12) ctrlSetText LLSTRING(paradropButton);
 };
 
 [{
+    params ["_vehicle", "_pfhID"];
+
     disableSerialization;
+
     private _display = uiNamespace getVariable QGVAR(menuDisplay);
-    if (isnil "_display") exitWith {
-        [_this select 1] call CBA_fnc_removePerFrameHandler;
+
+    if (isNil "_display") exitWith {
+        GVAR(interactionVehicle) = nil;
+        GVAR(interactionParadrop) = nil;
```

Comment:
This won't work, as `FUNC(startUnload)` requires both variables.

Reply:
Black magic is making it work regardless. Spooky.